lib/config/prettier.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  extends: ['plugin:shopify/esnext', 'prettier'],
+
+  plugins: ['prettier'],
+
+  rules: {
+    'prettier/prettier': [
+      'error',
+      {
+        singleQuote: true,
+        trailingComma: 'es5',
+        bracketSpacing: false,
+        jsxBracketSameLine: false,
+      },
+    ],
+
+    // rules to disable to prefer prettier
+    'shopify/binary-assignment-parens': 'off',
```

Comment:
Can you explain this more

Reply:
`shopify/binary-assignment-parens` expects something like this: 

```
const always = (config === 'always');
```

This will cause prettier to complain:
```
$ eslint . --max-warnings 0

/Users/ismailsyed/src/github.com/Shopify/eslint-plugin-shopify/lib/rules/class-property-semi.js
  14:20  error  Replace `(config·===·'always')` with `config·===·'always'`  prettier/prettier
```

Unfortunately, we'll have to turn this rule off in favor of the prettier experience.